pingcap/tidb
# *: make failpoint-disable before make clean

Signed-off-by: iosmanthus <myosmanthustree@gmail.com>


### What problem does this PR solve?

`make test` in TiDB and interrupt will cause some failpoint related files left in the working space. This pull request will try to clean these files before `make clean`.

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
	* make test and interrupt, then make clean to check all the generated/modified file are reverted.

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM

### Release note 

- make failpoint-disable before make clean
